configure.ac

```diff
@@ -134,6 +134,9 @@ AM_CONDITIONAL(HAVE_PAM, [test "x$with_pam" = "xyes"])
 
 PKG_CHECK_MODULES(DBUS, dbus-1)
 
+AC_CHECK_HEADER(acl/libacl.h,[],[AC_MSG_ERROR([Cannout find libacl headers. Please install libacl-devel])])
+AC_CHECK_HEADER(libmount/libmount.h,[],[AC_MSG_ERROR([Cannout find libmount headers. Please install libmount-devel])])
```

Comment:
libbtrfs AFAIS is already checked for. But the versioning here is ugly since older versions of btrfs did not include a version.h file.